media-sound/libopenmpt/libopenmpt-0.2.7386~beta20.3.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="OpenMPT based module player library and command-line player"
+DESCRIPTION="libopenmpt is a cross platform C++ and C libary to decode tracked \
+music files (modules) int a raw PCM audio stream. openmpt123 is a \
+cross-platform command line or terminal based module file player."
+HOMEPAGE="https://lib.openmpt.org/libopenmpt/"
+COPYRIGHT="2004-2016 OpenMPT contributors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${portVersion/\~/-}-autotools.tar.gz"
+CHECKSUM_SHA256="a6a7e6da1ae66e1cf46985ee92c182e50652d71b96135e9fa6048e132d844753"
+SOURCE_DIR="libopenmpt-${portVersion%~*}-autotools"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	libopenmpt$secondaryArchSuffix = $portVersion
+	lib:libopenmpt$secondaryArchSuffix = 0.0.20 compat >= 0
+	cmd:openmpt123 = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libmpg123$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libopenmpt${secondaryArchSuffix}_devel
+	devel:libopenmpt$secondaryArchSuffix = 0.0.20 compat >= 0
+	"
+REQUIRES_devel="
+	libopenmpt$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libz$secondaryArchSuffix
+	devel:libmpg123$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libFLAC$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
```

Comment:
I guess the recipe will also build without these, but since we are using the autotools-based tarball, I thought it could be a good idea to have them in the recipe.
This can be handy if someone want's to manually patch configure.ac and/or Makefile.am and then continue the build.
But if you think it is better to remove them that's OK for me.
Actually, I though it was good practice to add them because, if I'm not mistaken, configure looks for them even if it does not run them when the configure script is newer than configure.ac (and Makefile is newer than Makefile.am).